api/MediaStreamAudioSourceNode.json

```diff
@@ -29,10 +29,10 @@
             "version_added": "14"
           },
           "safari": {
-            "version_added": "6"
+            "version_added": false
```

Comment:
I suspect this change is incorrect and `MediaStreamAudioSourceNode` is supported. http://mdn-bcd-collector.appspot.com/tests/api/MediaStreamAudioSourceNode return false in many places making it hard to interpret the results.

@vinyldarkscratch can you check out why we're seeing a false result here?

Reply:
@vinyldarkscratch this conversation was resolved but it looks like the change still remains, so reopening.